lib/ecstatic.js

```diff
@@ -353,7 +353,7 @@ module.exports = function createMiddleware(_dir, _options) {
           if (req.statusCode === 404) {
             // This means we're already trying ./404.html and can not find it.
             // So send plain text response with 404 status code
-            status[404](res, next);
+            status[404](res);
```

Comment:
@jfhbrook: This might need an implicit next instead of `status[404]`?

e.g. I would suggest for *404 only*

```js
if (next) {
  next()
} else {
   status[404](res)
}
```